### PR TITLE
Feature doc read the doc

### PR DESCRIPTION
**Pull request by MWest2020**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | Not available
| License       | MIT
| Doc PR        | 

This is a redo of the Read the Docs for the Commonground Gateway

### CODE REVIEW

File: docs/features.md [hunk 1]
```diff
@@ -1,53 +1,71 @@
...
+
 ##NL Design System Ready
-The wab gateway provides an ease (and inline scripting save) way of providing application with NL Design System token files based on the organisation that is currently served (see multy-tenency). This allows applications to easily implement NL Design by using nl Design tokens (a full list of which can be viewed here). The only actions an application need to take is to include the following line before loading other ccs files `/api/organisation/{id}/nl-design-tokens.css`. Keep in mind that from a security perspective the API needs to be provided from the same domain as the actual application.
+
+The web gateway provides an easy (and inline scripting save) way of providing applications with NL Design System token files based on the organization that is currently served (see multy-tenancy). This allows applications to quickly implement NL Design using NL Design tokens (a complete list can be found here). The only action an application needs to take is to include the following line before loading other ccs files `/api/organisation/{id}/nl-design-tokens.css`. Security-wise, the API is to be provided from the same domain as the application.
```
Comment (rjzondervan) [inline docs/features.md:29]: @rubenvdlinde volgens mij moet dit `/api/organizations/{id}/nl-design-tokens.css` zijn?

File: docs/features.md [hunk 2]
```diff
@@ -1,53 +1,71 @@
...
+
+##Smart search
+
+The web gateway allows searching on any attribute marked as "searchable" even if the original API doesn't allow for searching that specific attribute or property. It also supports the searching of sub-entities way of dot annotation, e.g. `?field1.subfield1=searchme. All searchable fields of an endpoint are automatically collected for fuzzy search under the search query parameter allows you also to use `?search=searchme` be aware that this comes with a severe performance drain and that specific searches are almost always preferable. Wildcard search is only supported on string-type attributes.
```
Comment (rjzondervan) [inline docs/features.md:56]: Kleine aanpassing:

> The web gateway allows searching on any attribute marked as "searchable" even if the original API doesn't allow for searching that specific attribute or property. It also supports the searching of sub-entities way of dot annotation, e.g. `?field1.subfield1=searchme`. All searchable fields of an endpoint are automatically collected for fuzzy search under the search query parameter allows you also to use `?search=searchme` be aware that this comes with a severe performance drain and that specific searches are almost always preferable. Wildcard search is only supported on string-type attributes

**Review by rjzondervan — CHANGES_REQUESTED** [review]
Paar kleine puntjes, voornamelijk afkomstig uit eerdere foutjes

File: docs/features.md (hunk 2 above)
Comment (MWest2020) [reply docs/features.md]: Aangepast!

**Pull request closed without merging**